runtime/src/chpl-file-utils.c

```diff
@@ -158,6 +158,39 @@ qioerr chpl_fs_is_link(int* ret, const char* name) {
   return 0;
 }
 
+qioerr chpl_fs_is_mount(int* ret, const char* name) {
+  qioerr err = 0;
+  struct stat nBuf, parentBuf;
+  int exitStatus = 0;
+  size_t nameLen = strlen(name);
+  char* parent = (char* ) chpl_mem_allocManyZero(nameLen + 4, sizeof(char), CHPL_RT_MD_OS_LAYER_TMP_DATA, 0, 0);
+  strncat(parent, name, nameLen + 1);
+  strncat(parent, "/..", 3);
+  // TODO: Using "/" is not necessarily portable, look into this
```

Comment:
Think I might be inclined to do snprintf("%s/..", name) here, but in any case, if you don't do allocManyZero (above comment) then you'd need to change the first strncat() to str[n]cpy() instead. (So many ways to do the same thing ...)